.env

```diff
@@ -0,0 +1,5 @@
+DEVELOPMENT_DATABASE_URL="postgresql://Dana:test@localhost:5432/Lingobird_2"
```

Comment:
This file should never be on GitHub. Add it to `.gitignore`. But why is it needed on the front end in the first place?